theano/tensor/basic.py

```diff
@@ -6093,3 +6093,12 @@ def c_code_cache_version(self):
 
     def do_constant_folding(self, node):
         return False
+
+    def connection_pattern(self, node):
+        return [[False] for i in node.inputs]
+
+    def grad(self, inputs, grads):
+        return [DisconnectedType()() for i in inputs]
+
+    def R_op(self, inputs, eval_points):
+        return [zeros(inputs, self.dtype)]
```

Comment:
I already tried returning [None] (like Alloc does) but I got numerical errors in test_scan.py:T_Scan.test_rop2 and I did not understand why. I haven't tried modifying Alloc.R_op, though.